ipatests/test_webui/test_misc_cases.py

```diff
@@ -0,0 +1,41 @@
+#
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+#
+#
+# This program is free software; you can redistribute it and/or modify
```

Comment:
For new files, lets use the newer shorter code file header which is e.g. in `ipaserver/plugins/server.py` I.e. only the:

```
#
# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
#
```

Without the license related text below.